packages/d3fc-webgl/src/series/glLine.js

```diff
@@ -0,0 +1,180 @@
+import attributeBuilder from '../buffers/attributeBuilder';
+import glScaleBase from '../scale/glScaleBase';
+import programBuilder from '../program/programBuilder';
+import lineShader from '../shaders/line/baseShader';
+import drawModes from '../program/drawModes';
+import { rebind } from '@d3fc/d3fc-rebind';
+import uniformBuilder from '../buffers/uniformBuilder';
+import width from '../shaders/line/width';
+
+export default () => {
+    let program = programBuilder();
+    let xScale = glScaleBase();
+    let yScale = glScaleBase();
+    let decorate = () => {};
+
+    const xValueAttrib = 'aXValue';
+    const yValueAttrib = 'aYValue';
+    const xNextValueAttrib = 'aNextXValue';
+    const yNextValueAttrib = 'aNextYValue';
+    const xPrevValueAttrib = 'aPrevXValue';
+    const yPrevValueAttrib = 'aPrevYValue';
+    const cornerValueAttrib = 'aCorner';
+    const widthUniform = 'uWidth';
+    const screenUniform = 'uScreen';
```

Comment:
Would suggest you move these into an external file and export them so that they can be used in places like line.js:30